iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractThreeWindingsTransformerTest.java

```diff
@@ -456,8 +456,12 @@ public void validRatioStepArguments() {
         // Verify that other invalidPhaseStepArgument* tests are not throwing when arguments are ok
         ThreeWindingsTransformer transformer = createThreeWindingsTransformer();
         ThreeWindingsTransformer.Leg leg2 = transformer.getLeg2();
-        createRatioTapChangerOneStep(leg2, transformer.getTerminal(ThreeSides.TWO), 0.0, 0.0, 0.0,
+        try {
+            createRatioTapChangerOneStep(leg2, transformer.getTerminal(ThreeSides.TWO), 0.0, 0.0, 0.0,
                 0.0, 0.0);
+        } catch (Exception e) {
+            fail(e);
+        }
```

Comment:
Same comment for all the changes in this class: using `assertDoesNotThrow()` would be cleaner.